src/ui/zcl_abapgit_gui_page.clas.abap

```diff
@@ -120,7 +120,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE IMPLEMENTATION.
     ri_html->add( '<head>' ).
 
     ri_html->add( '<meta http-equiv="content-type" content="text/html; charset=utf-8">' ).
-    ri_html->add( '<meta http-equiv="X-UA-Compatible" content="IE=11,10,9,8" />' ).
+    ri_html->add( '<meta http-equiv="x-ua-compatible" content="IE=edge" />' ).
```

Comment:
I'm worried about this part, we did some evil tricks to get things working in sap gui

@sbcgua do you remember?

Reply:
I’m certainly not aware of all the history. Since you started a couple of SAPGUI release went out-of-maintenance. So not sure what assumptions we can make about the IE HTML control now-a-days

Reply:
Yeah, I remember there were some CSS issues, like tables sticking out of parent divs and IE11 behaved like IE9. So this should be tested well. Or maybe leave edge,11,10,9,8 ...

Reply:
yea, suggest leaving this line as is, then we can merge the other fixes